mu4/scores/view/scorethumbnail.h

```diff
@@ -0,0 +1,47 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_SCORES_SCORETHUMBNAIL_H
+#define MU_SCORES_SCORETHUMBNAIL_H
+
+#include <QQuickPaintedItem>
+#include <QPainter>
+
+namespace mu {
+namespace scores {
+
+class ScoreThumbnail : public QQuickPaintedItem
```

Comment:
Why register a separate QWidget component and not display pictures or their URLs in the model?

Reply:
URL is not available to me, because pictures from the archive.
The picture cannot just be transferred to qml.
Here one could use QQuickImageProvider, register images by url and in qml call by url.
But QQuickPaintedItem is most suitable for this task.